src/SecTester.Repeater/Bus/IRepeaterBusFactory.cs

```diff
@@ -2,5 +2,5 @@ namespace SecTester.Repeater.Bus;
 
 public interface IRepeaterBusFactory
```

Comment:
See https://github.com/NeuraLegion/sectester-js/pull/197#issuecomment-2168281090. Let's rename to  `IBridgeService`